static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain.js

```diff
@@ -16,6 +18,9 @@ type OutbrainDfpConditions = {
     useMerchandiseAdSlot: boolean,
 };
 
+const isInOutbrainTestingVariant = (): boolean =>
```

Comment:
We only use this once so maybe we can remove this function and change line 97 like so:

```js
const shouldTestOutbrainWidget: boolean = isInVariantSynchronous(commercialOutbrainTesting, 'variant');
```